src/test/codegen/loads.rs

```diff
@@ -122,7 +122,7 @@ pub fn borrow(x: &i32) -> &i32 {
 // CHECK-LABEL: @_box
 #[no_mangle]
 pub fn _box(x: Box<i32>) -> i32 {
-// CHECK: load {{(i32\*, )?}}i32** %x{{.*}}, !nonnull
+// CHECK: load {{(i32\*, )?}}i32** %x{{.*}}, align [[PTR_ALIGNMENT]]
```

Comment:
huh what happened here? we lost the nonnull attribute? We should be emitting it for references, too, right?

Reply:
There's no references here. It looks like we don't emit nonnull when loading the field of nonnull? That's really strange.

Reply:
Please open an issue for this

Reply:
Opened #97161